e2e/protractor/suites/actions/copy-move/destination-picker-dialog.test.ts

```diff
@@ -1,353 +0,0 @@
-      expect(await dataTable.isItemPresent(fileInDestination)).toBe(false, 'file is displayed');
-    });
-
-    it('[C263881] Folder links are not displayed', async () => {
-      await dialog.selectLocation('Personal Files');
-      await dataTable.doubleClickOnRowByName(destination);
-
-      expect(await dataTable.isItemPresent(folderInDestination)).toBe(true, `${folderInDestination} is not displayed`);
-      expect(await dataTable.isItemPresent(folder2InDestination)).toBe(true, `${folder2InDestination} is not displayed`);
-      expect(await dataTable.isItemPresent(folderLink)).toBe(false, 'Link to folder is displayed');
-    });
-
-    it('[C263885] User can see his Libraries', async () => {
-      await dialog.selectLocation('My Libraries');
-      expect(await dataTable.isItemPresent(site)).toBe(true, 'user site is not displayed');
-    });
-
-    it('[C263889] Search - No results displayed', async () => {
-      await dialog.searchFor('nonexistent-folder');
-
-      expect(await dataTable.isEmpty()).toBe(true, 'datatable not empty');
-      expect(await dataTable.getEmptyListText()).toEqual('No results found');
-    });
-
-    it('[C263888] Search - results found', async () => {
-      await dialog.searchFor(searchFolder);
-
-      expect(await dataTable.isItemPresent(searchFolder, username)).toBe(true, 'folder from Personal Files not displayed');
-      expect(await dataTable.isItemPresent(searchFolder, site)).toBe(true, 'folder from site not displayed');
-    });
-  });
-
-  describe('multiple selection', () => {
-    beforeAll(async () => {
-      await loginPage.loginWith(username);
-    });
-
-    beforeEach(async () => {
-      await page.dataTable.selectMultipleItems([file, destination]);
-      await page.toolbar.clickMoreActionsCopy();
-      await dialog.waitForDialogToOpen();
-    });
-
-    it('[C263879] Dialog title - multiple selection', async () => {
-      expect(await dialog.getDialogTitle()).toEqual(`Copy 2 items to...`);
-    });
-  });
-
-  describe('breadcrumb', () => {
-    beforeAll(async () => {
-      await loginPage.loginWith(username);
-    });
-
-    beforeEach(async () => {
-      await page.dataTable.selectItem(file);
-      await page.toolbar.clickMoreActionsCopy();
-      await dialog.waitForDialogToOpen();
-    });
-
-    it('[C263890] Personal Files breadcrumb - main node', async () => {
-      await dialog.selectLocation('Personal Files');
-      expect(await breadcrumb.currentFolder.getText()).toEqual('Personal Files');
-    });
-
-    it('[C263891] File Libraries breadcrumb - main node', async () => {
-      await dialog.selectLocation('My Libraries');
-      expect(await breadcrumb.currentFolder.getText()).toEqual('My Libraries');
-    });
-
-    it('[C263899] Search results breadcrumb', async () => {
-      await dialog.searchFor(searchFolder);
-      expect(await dialog.getToolbarTitle()).toEqual('Search results');
```

Comment:
getToolbarTitle function can be removed too

Reply:
Removed getToolbarTitle () function